mods/common/chrome/map-chooser.yaml

```diff
@@ -12,6 +12,19 @@ Background@MAPCHOOSER_PANEL:
 			Height: 20
 			Text: Choose Map
 			Font: Bold
+        Label@ORDERBY_LABEL:
```

Comment:
We use miniyaml, not regular yaml. It is different in a few ways, one of which is that instead of spaces we use tabs. Please follow this convention